src/main/java/redis/clients/jedis/JedisCluster.java

```diff
@@ -13,16 +13,37 @@ public class JedisCluster extends UnifiedJedis {
    * Default timeout in milliseconds.
    */
   public static final int DEFAULT_TIMEOUT = 2000;
+  /**
+   * Default amount of attemps for connecting.
+   */
   public static final int DEFAULT_MAX_ATTEMPTS = 5;
 
+  /**
+   * Creates a Jedis-Cluster Instance where only a single Host is being used as a "Cluster".<br>
```

Comment:
Please don't use `Jedis-Cluster`. You can use `JedisCluster` or `Cluster` or `cluster`.